tests/query_test.go

```diff
@@ -87,6 +87,16 @@ func (s *QuerySuite) TestQuery() {
 	s.Panics(func() {
 		s.Equal("bar", store.MustFindOne(queryErr).Foo)
 	})
+
+	foobar := newQueryFixture("foo bar")
```

Comment:
Initially, I tried to run `operators_test.go` inside of my forked repo folder, but I always get `can't load package: package github.com/jorbs/go-kallax: code in directory ~/go/src/github.com/jorbs/go-kallax expects import "gopkg.in/src-d/go-kallax.v1"`. I'm new on Go lang, but I think the packages are fixed to the original repo (github.com/src-d/go-kallax).